librz/include/rz_core.h

```diff
@@ -1274,11 +1274,9 @@ typedef bool (*RzCmdRegSync)(RzCore *core, RzRegisterType type, bool write);
 RZ_API bool rz_core_reg_assign_sync(RZ_NONNULL RzCore *core, RZ_NONNULL RzReg *reg, RzCmdRegSync sync_cb, RZ_NONNULL const char *name, ut64 val);
 RZ_API RZ_OWN RzList /*<RzRegItem *>*/ *rz_core_reg_filter_items_sync(RZ_NONNULL RzCore *core, RZ_NONNULL RzReg *reg, RzCmdRegSync sync_cb, RZ_NULLABLE const char *filter);
 
-/* rz_core_cmd replacement function declarations */
-// "? <expr>"
 RZ_API void rz_core_cmd_help_calc_expr(RZ_NONNULL RzCore *core, RZ_NONNULL const char *input);
-// a? or aaaaa like help message cases
 RZ_API void rz_core_cmd_show_analysis_help(RZ_NONNULL RzCore *core);
+RZ_API void rz_core_rtr_enable(RZ_NONNULL RzCore *core, RZ_BORROW const char *cmdremote);
```

Comment:
RZ_BORROW is only if the buffer is allocated by another function and held by a structure which should not be handled by the user (i.e. freed).

Reply:
Yes, please remove it